ios/FluentUI/Extensions/UIView+Extensions.swift

```diff
@@ -50,11 +50,11 @@ extension UIView {
         }
 
         if horizontally {
-            frame.origin.x = UIScreen.main.roundDownToDevicePixels(0.5 * (superview.frame.width - frame.width))
+            frame.origin.x = ceil(0.5 * (superview.frame.width - frame.width))
```

Comment:
Should we just use centerX/Y constraints instead?

Reply:
eventaully yes, we should get rid of all cetnerInsuperview function all together but that should be in a separate PR